singularity-eos/eos/eos_spiner_rho_sie.hpp

```diff
@@ -482,135 +514,161 @@ inline void SpinerEOSDependsRhoSie::calcBMod_(SP5Tables &tables) {
   }
 }
 
-inline SpinerEOSDependsRhoSie SpinerEOSDependsRhoSie::GetOnDevice() {
+template <template <class> class TransformerT>
+inline SpinerEOSDependsRhoSieTransformable<TransformerT>
+SpinerEOSDependsRhoSieTransformable<TransformerT>::GetOnDevice() {
   return STricks::GetOnDevice(this);
```

Comment:
Is the transformer trivially copyable? If so this should still be fine.

Reply:
I think it should be.